src/components/Moviecontainer/Moviecontainer.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import Movieitem from '../Movieitem/Movieitem'
+import './Moviecontainer.scss'
+import { Movie } from '../../common/types/movie'
+
+interface Props {
+    movieList: Movie[]
+}
+
+const Moviecontainer: React.FC<Props> = (props) => {
+
+  enum MovieTabs {
+    ALL = 'all',
+    DOCUMENTARY ='documentary',
+    COMEDY = 'comedy',
+    HORROR = 'horror',
+    CRIME = 'crime'
+  }
+
+  const tabs = [
+    {id: MovieTabs.ALL, label: 'All'},
+    {id: MovieTabs.DOCUMENTARY, label: 'Documentary'},
+    {id: MovieTabs.COMEDY, label: 'Comendy'},
+    {id: MovieTabs.HORROR, label: 'Horror'},
+    {id: MovieTabs.CRIME, label: 'Crime'},
+  ]
```

Comment:
Great update on this, like it! :D
A small idea still: You can spare some memory allocations, when you define such, "neverchanging" values outside of this component/function. The - otherwise not so harmful - problem with having these inside, is that they are redeclared every time, the component renders, taking newer and newer parts of the memory. We can save on these by moving them to, let's say a local `constants.ts` file.